app/lib/frontend/handlers.dart

```diff
@@ -53,3 +69,11 @@ shelf.Handler createAppHandler(shelf.Handler shelfPubApi) {
     return formattedNotFoundHandler(request);
   };
 }
+
+bool _shouldRedirectToPubDev(String path) {
+  if (path.startsWith('/api/')) return false;
+  if (path.endsWith('.tar.gz')) return false;
+  if (path.endsWith('.json')) return false;
```

Comment:
Why don't we redirect these? Can we maybe leave a comment at the end of each of these lines?